securedrop_client/queue.py

```diff
@@ -160,8 +159,13 @@ def logout(self) -> None:
     def login(self, api_client: API) -> None:
         logger.debug('Passing API token to queues')
 
-        # Setting realistic (shorter) timeout for general requests so that user feedback
-        # is faster
+        # Setting realistic (shorter) timeout for general requests so that user feedback is faster.
+        # General requests includes:
+        #       FileDownloadJob
+        #       MessageDownloadJob
+        #       ReplyDownloadJo
```

Comment:
for `FileDownloadJob, MessageDownloadJob, ReplyDownloadJob` we're using `DownloadJob._get_realistic_timeout` so we might want to remove them from this list to prevent confusion

Reply:
good point, removed comment